addons/web/static/tests/core/commands/command_palette_tests.js

```diff
@@ -1490,3 +1491,41 @@ QUnit.test("remove namespace with backspace", async (assert) => {
     await nextTick();
     assert.strictEqual(target.querySelector(".o_command_palette .o_namespace").innerText, "@");
 });
+
+QUnit.test("generate new session id when opened", async (assert) => {
+    assert.expect(4);
+
+    let lastSessionId;
+    CommandPalette.lastSessionId = 0;
+    testComponent = await mount(TestComponent, { env, target });
```

Comment:
in 15.3 the mount() util API has changed:
`mount(TestComponent, target, { env })`